package.json

```diff
@@ -12,6 +12,10 @@
         "branch": "",
         "SHA": ""
     },
+    "dependencies": {
```

Comment:
I also meant NPM, But for eslint, could remove dev/testing dependencies from package.json if the install size is too big.
It is possible to put the node_modules folder in the extension directory right?